content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "3D Terrain Generation with Perlin Noise in Processing",
+  "description": "In this coding challenge, I create a 3D procedural terrain using Perlin Noise and the beginShape() function in Processing.",
+  "videoNumber": "11",
+  "videoId": "IKB1hWWedMk",
+  "date": "2016-05-04",
+  "languages": ["Processing"],
+  "topics": ["perlin noise", "terrain generation", "procedural"],
+  "canContribute": true,
+  "relatedChallenges": [],
+  "timestamps": [
+    { "time": "00:00", "title": "Welcome! Introduction to the Challenge" },

```

Comment:
I think for timestamps on the site we're just doing `Introduction to the Challenge`, `What do we need to do?` etc. without the `Welcome!` or `Explain!` part

Reply:
Thanks for the suggestions, will keep this in mind for the future.

Reply:
Thanks for raising this @klinegareth! I've gone through a variety of different styles fo the timestamps. I think I've settled on preferring timestamps that are concise and to the point so even just "Introduction" would work for `00:00`. Open to your ideas and no need to make these changes now!